npm/react/examples/using-babel-typescript/README.md

```diff
@@ -31,14 +31,14 @@ npm test
 
 ## Specs
 
-See spec files [src/\*.spec.js](src). The specs are bundled using [babel.config.js](babel.config.js) settings via [cypress/plugins/index.js](cypress/plugins/index.js) file that includes file preprocessor.
+See spec files [src/\*.spec.js](src). The specs are bundled using [babel.config.js](babel.config.js) settings via [cypress/plugins/index.js](cypress/plugins/index.js) file that includes file setupDevServer.
 
 ```js
 // let's bundle spec files and the components they include using
 // the same bundling settings as the project by loading .babelrc
-const preprocessor = require('@cypress/react/plugins/babelrc')
+const setupDevServer = require('@cypress/react/plugins/babelrc')
```

Comment:
Is `'@cypress/react/plugins/babelrc'` a typo? I don't see a "babelrc" dev server, just a "babel" dev server.

Reply:
it is indeed a typo, old as time

Reply:
fixed in 38f56aa3b71493013e6bb94cb345d48a0c013f26